cms/static/cms/js/modules/cms.base.js

```diff
@@ -324,6 +333,7 @@ CMS.API.Helpers = {
      * @param {String[]} [params] array of `param=value` strings to update the url
      * @returns {String}
      */
+    // TODO get rid of this
```

Comment:
we should also remove the `once`, `debounce` and `throttle` as we now have lodash and lodash provides those.